src/plugins/sails.js

```diff
@@ -171,6 +171,9 @@ io.socket.on('game', function (evData) {
         case 'reLogin':
           store.dispatch('updateGameThenResetPNumIfNull', evData.data.game);
           break;
+        case 'spectatingUsers':
+          store.dispatch('updateGameThenResetPNumIfNull', evData.data.game);
+          break;
```

Comment:
These can likely be combined since they're the same.
```
        case 'reLogin':
        case 'spectatingUsers':
          store.dispatch('updateGameThenResetPNumIfNull', evData.data.game);
          break;
```